include/inviwo/core/datastructures/datasequence.h

```diff
@@ -125,6 +125,10 @@ class DataSequence {
 
     void erase(iterator pos) { data_.erase(pos.base()); }
     void erase(const_iterator begin, const_iterator end) { data_.erase(begin.base(), end.base()); }
+
```

Comment:
I wonder if the decltype(getData) above is the reason for all the warnings about internal linkage below...
The lambda might have unique type in each TU. which would be bad.
Maybe you can try and make those into a named struct with a call operator?